.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -83,6 +84,6 @@ apache/pulsar CI based on GitHub Actions has constrained resources and quota.
 GitHub Actions provides separate quota for pull requests that are executed in 
 a forked repository.
 
-The tests will be run in the forked repository until all PR review comments have
+The tests will be run in the forkedrepository until all PR review comments have
```

Comment:
Unintended changes?

Reply:
Good catch, I misread the diff initially thinking it was a fix.

Reply:
Thanks for catching the mistake